README.md

```diff
@@ -1,2 +1,61 @@
 # Newsweek
 This project demonstrates the use of Twitter’s Bootstrap framework by building a mock-up of Newsweek site that actually uses it
+
+> The structure of the index page (index.html) is based on the Newsweek.com main page
```

Comment:
- [ ] Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...Check this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more info.

Reply:
Improved the README file